prototype2/multigrid/mesytec/test/BuilderReadoutsTest.cpp

```diff
@@ -119,7 +119,7 @@ TEST_F(BuilderReadoutsTest, t00004) {
   EXPECT_EQ(ShortestPulsePeriod, 266662);
   EXPECT_EQ(external_triggers, 467);
   EXPECT_EQ(invalid_plane, 0);
-  EXPECT_EQ(plane_counts.size(), 18);
+  ASSERT_EQ(plane_counts.size(), 18);
```

Comment:
This seems like an arbitrary modification. Why change this one to ASSERT and not the others? (comment also holds for
the other five occurrences).

Reply:
Because the call to `size()` is followed by accessing of the elements in that vector. If the size is less than 18,  this can/will result in a crash due to out of bounds emory access. This happened on my machine.